app/app/static/custom/ourscript.js

```diff
@@ -0,0 +1,117 @@
+var timers = [];
+var timerIntervals = [];
+
+function showAddToQueue() {
+	$('#addToQueue').css('display', 'block');
+}
+
+function hideAddToQueue() {
+	document.getElementById('addToQueueForm').reset();
+	$('#addToQueue').css('display', 'none');
+}
+
+$('#addToQueueForm').submit(function (e) {
+	e.preventDefault();
+
+	let name = document.getElementById('studentName').value;
+	let id = document.getElementById('studentNumber').value;
+	let unit = document.getElementById('unitCode').value;
+	let team = document.getElementById('team').value;
+	let enquiry = document.getElementById('enquiryType').value;
```

Comment:
be consistent of the variables, if the "elementId" is "enquiryType" then the variable should be "enquiryType".

eg.
instead of just saying `let name = document.getElementById('studentName').value;` , it should be `let studentName= document.getElementById('studentName').value;`

it makes it easier for other developer to easily see which variables corresponds to other variables instead of having "synonyms"